core-tests/src/test/java/org/mule/runtime/core/internal/streaming/bytes/PoolingByteBufferManagerTestCase.java

```diff
@@ -92,15 +91,15 @@ public void differentPoolsPerCapacity() throws Exception {
   }
 
   @Test
-  public void capacity() throws Exception {
+  public void capacity() {
     assertCapacity(CAPACITY);
     assertCapacity(OTHER_CAPACITY);
   }
 
   @Test
   public void limitTotalMemory() throws Exception {
     final long maxMemory = 40L;
-    final int bufferCapacity = toIntExact(maxMemory / 4);
+    final Long bufferCapacity = round(maxMemory * MAX_STREAMING_PERCENTILE) / 2;
```

Comment:
you can use the primitive here instead of the wrapper

Reply:
no, because I need the `.intValue()` later on